challenges/classes.js

```diff
@@ -1,9 +1,36 @@
 // Copy and paste your prototype in here and refactor into class syntax.
 
+class CuboidMaker {
+  constructor(properties) {
+    this.length = properties.length;
+    this.width = properties.width;
+    this.height = properties.height;
+  }
+  volume (){
+    return (this.length * this.width * this.height);
+  }
+  surfaceArea (){
+    return (2*(length * (width + length) * (height + width) * height));
```

Comment:
There are 2 issues in the return statement for suraceArea:
1. What is length, width, and height referring to? What is missing?
2. The logic is currently `2 * (l * (w + l) * (h + w) * h)` when it should be `2 * (lw) + (lh) + (wh)`